airflow-core/src/airflow/ui/public/i18n/locales/en/common.json

```
@@ -91,7 +91,8 @@
     "defaultMessage": "An unexpected error occurred",
     "home": "Home",
     "notFound": "Page Not Found",
-    "title": "Error"
+    "title": "Error",
+    "authenticating": "Authenticating..."
```

Should we be a bit more verbose on this what it means? Like for example:
```suggestion
    "title": "Authentication Error",
    "authenticating": "Authentication not accepted, forwarding to login page..."
```

---

Then issue itself is not that the auth is not accepted but rather that the auth flow is delay or unsynced so that it displays an ugly ui bug whilst the auth flow is finishing. Hence I think "Authenticating..." is more accurate.